mp/src/game/shared/momentum/fx_mom_shared.cpp

```diff
@@ -178,16 +180,42 @@ void FX_FireBullets(int iEntIndex, const Vector &vOrigin, const QAngle &vAngles,
         lagcompensation->StartLagCompensation(pPlayer, pPlayer->GetCurrentCommand());
 #endif
 
-    for (int iBullet = 0; iBullet < pWeaponInfo->m_iBullets; iBullet++)
-    {
-        RandomSeed(iSeed); // init random system with this seed
+    // TF2 spread pattern
+    Vector vecFixedPattern[] = {
```

Comment:
This can be a static const `g_vecFixedPattern` outside of this method, you don't want it to be declared every time the function is called if it's not going to change.